packages/attribute-slicer-powerbi/src/AttributeSlicerVisual.ts

```diff
@@ -424,12 +391,31 @@ export default class AttributeSlicer extends VisualBase {
      * Listener for when loading more data
      */
     private onLoadMoreData(item: any, isSearch: boolean) {
+        const loadMoreData = () => {
```

Comment:
Is one of these branches for the legacy api and no longer needed here?

Reply:
Sort of, it is a fallback.  Since they don't have the `loadMoreData` function in their new API, this falls back to using a workaround to loadMoreData (just uses the host object off of the selection manager)